### PR TITLE
chore(#8990): adds .gitattributes file

### PR DESCRIPTION
**Pull request by fardarter**
Addresses issue: https://github.com/medic/cht-core/issues/8990

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+*.pdf binary
+*.png binary
+*.psd binary
+*.webp binary
+
+# Fonts
+*.woff2 binary
+
+# Other
+*.exe binary
```
Comment (garethbowen) [inline .gitattributes:57]: I think we can drop this. Git is smart enough to work this out and means we don't need to manage it. As your linked blog post says...

> ...relying on Git to auto-detect binary files (yes Git is clever enough to do that)

**Review by garethbowen — CHANGES_REQUESTED** [review]
Some simplification recommended but the line ending stuff is great.

File: .gitattributes (hunk 1 above)
Comment (fardarter) [reply .gitattributes:57]: We've seen xlsx get diffed on our repo, which is why I lost trust in the auto detect a little. Including it in the file solved that, but will make the change regardless.

Comment (fardarter) [reply .gitattributes]: @garethbowen have made the change

**Review by garethbowen — APPROVED** [review]
Lovely!

**Pull request merged**